pkgs/development/octave-modules/symbolic/default.nix

```diff
@@ -0,0 +1,59 @@
+{ buildOctaveLibrary
+, stdenv
+, fetchurl
+, octave
+, gnuplot
+, python
+, python2Packages
+, texinfo
+}:
+
+let
+  # Need to use sympy 1.5.1 for https://github.com/cbm755/octsympy/issues/1023
+  # It has been addressed, but not merged yet.
+  # In the meantime, we create a Python environment with Python 3, its mpmath
+  # version and sympy 1.5 from python2Packages.
+  pythonEnv = (let
+      overridenPython = let
+        packageOverrides = self: super: {
+          sympy = super.sympy.overridePythonAttrs (old: rec {
+            version = python2Packages.sympy.version;
+            src = python2Packages.sympy.src;
+          });
+        };
+      in python.override {inherit packageOverrides; self = overridenPython; };
```

Comment:
Correct. I have a branch that I am currently working on, where there is a `wrap.sh` helper script that is used during environment creation. On that branch, I identified the issue and fixed it (quite literally last night).

This will be fixed when I merge that particular branch back into this PR branch.